compiler-rt/test/fuzzer/features_dir.test

```diff
@@ -1,5 +1,5 @@
 # Tests -features_dir=F
-# REQUIRES: linux
```

Comment:
Why do we no longer require linux? This doesn't sound like it's intentional in this CL.

Reply:
Yes, that was a mistake. I've added it back.